plugins/node/opentelemetry-instrumentation-redis/test/redis.test.ts

```diff
@@ -133,26 +133,30 @@ describe('redis@2.x', () => {
       description: string;
       command: string;
       args: string[];
+      serializedArgs: Array<string>;
```

Comment:
The name `serializedArgs` is a bit vague. 
WDYT about calling it `expectedDbStatement`, and storing it as text instead of array which we then need to `join(' ')`?
Might make the test a bit simpler to read and understand IMO.

Reply:
Sure. I fixed it also in ioredis.